axum/src/json.rs

```diff
@@ -167,6 +143,47 @@ impl<T> From<T> for Json<T> {
     }
 }
 
+impl<T> Json<T>
+where
+    T: DeserializeOwned,
+{
+    /// Construct a `Json<T>` from `&Bytes`. Most users should prefer to use the `FromRequest` impl
+    /// but special cases may require first extracting a `Request` into `Bytes` then optionally
+    /// constructing a `Json<T>`.
+    ///
+    /// An example where this would be useful would be one in which a service needs to pass through
+    /// or otherwise preserve the exact byte representation of the request while also interrogating
+    /// it for metadata that may be useful in determining exactly what to do with the preserved
+    /// bytes.
```

Comment:
I'm not sure but I kinda feel like we should remove this paragraph. I worry it might be confusing or potentially misleading.

Reply:
Okay, fair enough.